source/_layouts/default.html

```diff
@@ -55,7 +55,7 @@
 
     <div id="doc-switchers" class="container">
       <div class="col-md-8">{% include template/doc_switcher.html %} {% include template/version_switcher.html %}</div>
-      <div class="col-md-4"></div>
+      <div class="col-md-4"><div class="visible-sm-inline visible-xs-inline"><a href="#document-nav">Jump to table of contents</a></div></div>
```

Comment:
Hmmmmmmm. Yeah, that's all super feasible. But we'll also need an option to use when we're not in a named product manual, since there's still a sidebar nav to jump to. 

(Bah, the double-nav problem comes back to bite us in the butt again.)

Do you think it would be simpler to refer to it as the "sidebar navigation?" I realize that's not strictly true on mobile (which is the whole problem here), but I wonder if that would actually be the least confusing approach.